### ISSUE TITLE
Goal "start" is rebuilding the image 

### ISSUE THREAD

**Issue by eemmiirr** [alexec/docker-maven-plugin#45]
When "start" goal is run it's triggering the rebuild of the image even the image exists in the local images.

**Comment by alexec** [comment]
Ah - does this occur when you do start twice in a row?

**Comment by eemmiirr** [comment]
It occurs when i do just run mvn clean install with the following config. When i rerun it again it doesn't take the prebuild image it builds it again

<plugin>
                        <groupId>com.alexecollins.docker</groupId>
                        <artifactId>docker-maven-plugin</artifactId>
                        <version>2.4.0</version>
                        <configuration>
                            <host>http://localhost:2375</host>
                            <cache>true</cache>
                            <removeIntermediateImages>false</removeIntermediateImages>
                        </configuration>
                        <executions>
                            <execution>
                                <id>start</id>
                                <phase>pre-integration-test</phase>
                                <goals>
                                    <goal>start</goal>
                                </goals>
                            </execution>
                            <execution>
                                <id>stop</id>
                                <phase>post-integration-test</phase>
                                <goals>
                                    <goal>stop</goal>
                                </goals>
                            </execution>
                        </executions>
                    </plugin>

**Comment by alexec** [comment]
Hmmm - any chance you can provide the source code?

**Comment by eemmiirr** [comment]
The docker image i am trying to build?

**Comment by alexec** [comment]
Basically, I don't know what the issue is, so I'd like to get more information to diagnose it. Perhaps the easiest thing would be to paste the command line output?

**Comment by eemmiirr** [comment]
It seems like it builds first the image and its ok. It tags is as test/redis-cluster:3.0.0 but then when it get to the start goal it does:
```
[INFO] Docker version 1.5.0
[INFO] 3 * Sending client request on thread main
3 > GET http://localhost:2375/images/null%2Fuser-service_redis/json
3 > Accept: application/json

```

**Comment by alexec** [comment]
*Should be* fixed in 2.4.1.
